torch_xla/csrc/runtime/BUILD

```diff
@@ -237,7 +237,7 @@ cc_library(
     deps = [
         ":debug_macros",
         ":sys_util",
-        "@tsl//tsl/distributed_runtime/preemption:preemption_sync_manager",
+        "@xla//xla/tsl/distributed_runtime/preemption:preemption_sync_manager",
         "@xla//xla/pjrt/distributed",
```

Comment:
Thanks for catching this change to accept OpenXLA's `moving TSL to XLA` change, @yeounoh!